src/webxdc/webxdc_tests.rs

```diff
@@ -1789,6 +1789,23 @@ async fn test_webxdc_delete_event() -> Result<()> {
     Ok(())
 }
 
+#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+async fn test_chat_delete() -> Result<()> {
+    let alice = &TestContext::new_alice().await;
+    let chat_id = create_group_chat(alice, ProtectionStatus::Unprotected, "foo").await?;
+    let instance = send_webxdc_instance(alice, chat_id).await?;
+    chat_id.delete(alice).await?;
+    let EventType::WebxdcInstanceDeleted { msg_id } = alice
+        .evtracker
+        .get_matching(|evt| matches!(evt, EventType::WebxdcInstanceDeleted { .. }))
+        .await
+    else {
+        unreachable!();
+    };
+    assert_eq!(msg_id, instance.id);
+    Ok(())
+}
+
```

Comment:
would make sense to also make a test that checks that `WebxdcInstanceDeleted` is only emitted for deleted webxdc messages and one that makes sure only that it only emits the events from the chat where the message was deleted from.

Reply:
I extended the test, now it creates two chats with webxdc messages and also sends a text message to the chat aimed for deletion